lib/person_cards.rb

```diff
@@ -1,14 +1,16 @@
 # frozen_string_literal: true
 
 class PersonCard
-  attr_reader :proxy_image, :memberships
-
-  # TODO: pass fewer arguments!
-  def initialize(person:, proxy_image:, memberships:, top_identifiers:)
+  def initialize(person:, term:)
     @person          = person
-    @memberships     = memberships # should be able to get from Person
-    @proxy_image     = proxy_image # get from Legislature
-    @top_identifiers = top_identifiers # get from Legislature
+    @term            = term
+    @legislature     = term.legislature
+    @top_identifiers = term.top_identifiers
```

Comment:
NB: this is now better documented: https://github.com/everypolitician/viewer-sinatra/blob/master/CONTRIBUTING.md